consensus/byzantine_test.go

```diff
@@ -594,3 +594,7 @@ func (br *ByzantineReactor) Receive(e p2p.Envelope) {
 	br.reactor.Receive(e)
 }
 func (br *ByzantineReactor) InitPeer(peer p2p.Peer) p2p.Peer { return peer }
+
+// func (br *ByzantineReactor) QueueUnprocessedEnvelope(e p2p.UnprocessedEnvelope) {
```

Comment:
why was this commented?

Reply:
ill remove this, it was for testing but then wasnt sure if i should try to add it back. there was an issue with some tests that i couldnt figure out the issue